visual-dotnet/SauceLabs.Visual/BuildFactory.cs

```diff
@@ -142,6 +142,10 @@ private static async Task<VisualBuild> Create(VisualApi api, CreateBuildOptions
             }
 
             options.CustomId ??= EnvVars.CustomId;
+            options.Name ??= EnvVars.BuildName;
```

Comment:
Just to be sure, aren't we expecting the EnvVar to take precedence over set vars ?

Reply:
Oopsie. Should be fixed now